routes/clients.js

```diff
@@ -195,6 +195,43 @@ router.delete('/:idClient/products/:idProduct/favorites', (req, res) => {
             res.sendStatus(200)
         }
     })
+});
+
+/////////////////////////////////////////////////////// Favorites shops //////////////////////////////////////////////////////////
+
+//GET all the favorite shops of one client by it's ID
+router.get('/:idClient/favorites/shops', (req, res) => {
+    connection.query('SELECT * FROM favorite_shop AS f JOIN shop AS s ON f.id_shop = s.id WHERE id_client = ?', req.params.idClient, (err, result) => {
+        if (err) {
+            res.status(500).json(err)
```

Comment:
Il vaut mieux écrire un texte pour avoir en clair ce qui n'a pas fonctionné.
```suggestion
            res.status(500).json("Nous n'avons pas pu récupérer les données en favoris du client")
```